corehq/apps/export/views.py

```diff
@@ -2104,9 +2124,10 @@ class DownloadNewFormExportView(GenericDownloadNewExportMixin, DownloadFormExpor
     urlname = 'new_export_download_forms'
     filter_form_class = EmwfFilterFormExport
     export_filter_class = SubmitHistoryFilter
+    export_class = FormExportInstance
 
     def _get_export(self, domain, export_id):
-        return FormExportInstance.get(export_id)
+        return get_exports(domain, [export_id], self.export_class)[0]
```

Comment:
Is using `get_exports` necessary?  Couldn't this just be:
```python
return self.export_class.get(export_id)
```
Might be able to move that to the superclass too, since Forms, Cases, and SMSs have the same implementation. I haven't verified that, though.